spec/unit/installer_spec.rb

```diff
@@ -86,21 +89,19 @@ def @installer.clean_pod_sources
       end
 
       it 'in runs the post-install hooks before serializing the Pods project' do
-        @installer.stubs(:prepare_pods_project)
         @installer.stubs(:run_podfile_pre_install_hooks)
-        @installer.stubs(:install_file_references)
-        @installer.stubs(:install_libraries)
-        @installer.stubs(:set_target_dependencies)
         @installer.stubs(:write_lockfiles)
         @installer.stubs(:aggregate_targets).returns([])
         @installer.unstub(:generate_pods_project)
-        def @installer.run_podfile_post_install_hooks
-          @hook_called = true
-        end
+        generator = @installer.send(:create_generator)
+        @installer.stubs(:create_generator).returns(generator)
+        generator.stubs(:generate!)
+        generator.stubs(:share_development_pod_schemes)
+
+        hooks = sequence('hooks')
```

Comment:
I changed this to use Mocha sequences because the old way doesn't work when stubbing two classes